test/libsolidity/util/BytesUtils.cpp

```diff
@@ -252,7 +252,35 @@ string BytesUtils::formatBytes(
 		if (*_bytes.begin() & 0x80)
 			os << formatSigned(_bytes);
 		else
-			os << formatUnsigned(_bytes);
+		{
+			std::string hex(formatHex(_bytes));
+			unsigned int value = u256(_bytes).convert_to<unsigned int>();
+			if (value < 10)
+				os << dec;
+			else if (value >= 10 && value <= 0xff) {
```

Comment:
I very much prefer `10` over `0xa` :)

Reply:
yeah me too ;) that was a typo :)

Reply:
ahh.. I changed that now to 0x10.. but probably you're right